crates/fuel-core/src/service/adapters/graphql_api.rs

```diff
@@ -144,6 +144,14 @@ impl DatabaseMessages for Database {
             .map(|result| result.map_err(StorageError::from))
             .into_boxed()
     }
+
+    fn message_is_spent(&self, nonce: &Nonce) -> StorageResult<bool> {
```

Comment:
Maybe rename this to `is_message_spent` for consistency (or change `is_message_spent` to `message_is_spent`)